plugin.go

```diff
@@ -59,6 +68,11 @@ func (p *BasePlugin) ApplyMiddlewarePrivateMux(h http.Handler) http.Handler {
 	return h
 }
 
+// WrapGraphQLClientTransport wraps the http.RoundTripper used for GraphQL requests.
+func (p *BasePlugin) WrapGraphQLClientTransport(transport http.RoundTripper) http.RoundTripper {
```

Comment:
I might have suggested naming this something like `WrapOriginTransport` because it's on the origin-server-side of a Bramble instance.